### DISCUSSION TITLE
Directory support in FileInput and Dropzone

### DISCUSSION THREAD

**Discussion by ponchofiesta** [mantinedev/mantine#3476] [Feature requests]
I'm evaluating to use Mantine for a desktop app based on Tauri. Actually I can't open a directory using Mantine.

AFAIK HTML/Javascript does support it:

```html
<input type="file" webkitdirectory directory multiple/>
```

```javascript
const dirHandle = await window.showDirectoryPicker();
```

I don't need access to the directory via Javascript. I only need the path which was chosen.

**Comment by rtivital** [comment]
I'll add option to set additional props to the file input in FileInput component in next patch.
